pkg/nificlient/parametercontext.go

```diff
@@ -34,7 +34,7 @@ func (n *nifiClient) GetParameterContext(id string) (*nigoapi.ParameterContextEn
 	}
 
 	// Request on Nifi Rest API to get the parameter context informations
-	pcEntity, rsp, body, err := client.ParameterContextsApi.GetParameterContext(context, id)
+	pcEntity, rsp, body, err := client.ParameterContextsApi.GetParameterContext(context, id, nil)
```

Comment:
There is a query parameter to include inherited parameter contexts. Should that be set here instead of sending `nil`?

https://github.com/konpyutaika/nigoapi/blob/master/pkg/nifi/api_parameter_contexts.go#L472-L474

Reply:
I can only set it to `false`. 
If I set it to `true`, the values returned will not be those associated with the parameter context but the ones "calculated" after the inheritance.

> Whether or not to include inherited parameters from other parameter contexts, and therefore also overridden values. If true, the result will be the 'effective' parameter context.